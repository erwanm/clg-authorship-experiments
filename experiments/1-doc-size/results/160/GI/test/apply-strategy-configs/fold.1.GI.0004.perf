0.506220	0.767000	0.660000
