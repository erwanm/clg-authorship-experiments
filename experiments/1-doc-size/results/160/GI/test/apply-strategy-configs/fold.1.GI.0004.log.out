apply-config.sh: applying config '/tmp/expe1/160/process//selected-meta-configs/fold.1.strategy-as-meta.GI.0004.model/strategy-configs/fold.1.GI.0004' to cases '/home/moreaue/dump/expe1//results/160/GI/test/apply-strategy-configs/only-cases.tmp.Fv7m063X'; results written to '/home/moreaue/dump/expe1//results/160/GI/test/apply-strategy-configs/fold.1.GI.0004'
apply-config.sh: done.
